pkg/features/pilot/pilot.go

```diff
@@ -70,6 +70,14 @@ var (
 	// Default is 10s, Example: "300ms", "10s" or "2h45m".
 	DebounceMax = env.RegisterDurationVar("PILOT_DEBOUNCE_MAX", 10*time.Second, "").Get()
 
+	EnableEDSDebounce = env.RegisterBoolVar(
+		"PILOT_ENABLE_EDS_DEBOUNCE",
+		true,
+		"If enabled, Pilot will include EDS pushes in the push debouncing, configured by PILOT_DEBOUNCE_AFTER and PILOT_DEBOUNCE_MAX."+
+			" This means that EDS pushes may be delayed, but there will be fewer pushes."+
+			" By default this is disabled, meaning any endpoint change will trigger an EDS push.",
```

Comment:
IMO we don't want this behavior by default.  Further, it appears this text is in conflict with the implementation.

Reply:
Good catch. I initially was going to put it on by default but decided that wasn't a good idea and forgot to update the doc